### PR TITLE
BRIDGE-1068 Change the medication tracking to include two step question about timing

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Modifications to the hardcoded logic to allow for two questions about medication timing.

### CODE REVIEW

File: mPowerSDK/TasksAndSteps/MedicationTracker/APHMedicationTrackerDataStore.m [hunk 1]
```diff
@@ -220,7 +228,7 @@ - (BOOL)hasChanges {
 - (void)commitChanges {
 
     // store the moment in day result in memeory
-    ORKStepResult *momentInDayResult = [self.changesDictionary objectForKey:kMomentInDayResultKey];
+    id momentInDayResult = [self.changesDictionary objectForKey:kMomentInDayResultKey];
```
Comment (Erin-Mounts) [inline mPowerSDK/TasksAndSteps/MedicationTracker/APHMedicationTrackerDataStore.m]: can this be of a type other than NSArray<ORKStepResult *> *?

Comment (Erin-Mounts) [inline mPowerSDK/TasksAndSteps/MedicationTracker/APHMedicationTrackerDataStore.m]: stupid thing removed angle brackets as html tags

Comment (Erin-Mounts) [inline mPowerSDK/TasksAndSteps/MedicationTracker/APHMedicationTrackerDataStore.m]: NSArray<ORKStepResult *> *

Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/MedicationTracker/APHMedicationTrackerDataStore.m]: No, changed it

**Comment by Erin-Mounts** [comment]
Looks good

**Pull request merged**